kafka-ui-react-app/src/components/Topics/New/__test__/New.spec.tsx

```diff
@@ -90,12 +92,42 @@ describe('New', () => {
     await act(() => {
       userEvent.type(screen.getByPlaceholderText('Topic Name'), topicName);
     });
+    await act(() => {
+      userEvent.type(
+        screen.getByPlaceholderText('Number of partitions'),
+        minValue
+      );
+    });
+    await act(() => {
+      userEvent.type(
+        screen.getByPlaceholderText('Min In Sync Replicas'),
+        minValue
+      );
+    });
+    await act(() => {
+      userEvent.type(
+        screen.getByPlaceholderText('Replication Factor'),
+        minValue
+      );
+    });
+    await act(() => {
+      userEvent.type(
+        screen.getByPlaceholderText('Time to retain data (in ms)'),
+        retainTime
+      );
+    });
+    await act(() => {
+      userEvent.type(
+        screen.getByPlaceholderText('Maximum message size'),
+        minValue
+      );
+    });
```

Comment:
`
await userEvent.type(
      screen.getByLabelText('Maximum message size in bytes *'),
      minValue
    );
`

Reply:
after you can add this 
`expect(screen.getByText('Create topic')).toBeEnabled();`